src/mrinufft/operators/base.py

```diff
@@ -225,6 +226,42 @@ def with_off_resonnance_correction(self, B, C, indices):
         from ..off_resonnance import MRIFourierCorrected
 
         return MRIFourierCorrected(self, B, C, indices)
+    
+    def compute_smaps(self, method=None):
+        """Compute the sensitivity maps and set it.
+
+        Parameters
+        ----------
+        method: callable or dict or array
+            The method to use to compute the sensitivity maps.
+            If an array, it should be of shape (NCoils,XYZ) and will be used as is.
+            If a dict, it should have a key 'name', to determine which method to use.
+            other items will be used as kwargs.
+            If a callable, it should take the samples and the shape as input.
+            Note that this callable function should also hold the k-space data 
+            (use funtools.partial)
+        """
+        if isinstance(method, np.ndarray):
+            self.smaps = method
+            return None
+        if not method:
+            self.smaps = None
+            return None
+        kwargs = {}
+        if isinstance(method, dict):
+            kwargs = method.copy()
+            method = kwargs.pop("name")
+        if isinstance(method, str):
+            method = get_smaps(method)
+        if not callable(method):
+            raise ValueError(f"Unknown smaps method: {method}")
+        self.smaps, self.SOS = method(
```

Comment:
Mention that you have the SOS available as well in this case.

Reply:
Where do I mention though?